docs/sources/tutorials/stack_guide_graphite.md

```diff
@@ -17,7 +17,7 @@ database has yet to match Graphite's query flexibility and analytics potential.
 Graphite has a reputation for being tricky to install and scale. This guide aims to show
 that is not really the case, or, at least, that it is a lot better than you expect.
 
-> This guide does not only aim to be only be an install guide but to also teach you
+> This guide does not only aim to be an install guide but to also teach you
```

Comment:
Please change to: This is an install guide that also teaches you of the mechanics of metric collection ...

Reply:
@achatterjee-grafana Committed the suggested changes. Please approve